### PR TITLE
some tweaks to users and menus got tablet

### PR DESCRIPTION
**Pull request by druiz17**
Test 1
 - Click users on use on the tablet and the home button at the bottom should change.

Test 2 
- Click menu on the tablet then click on a submenu, then click the menu name at the top, which brings you back to the previous menu( in this test the main menu). The name on top should say "Menu" not 
"root"

### CODE REVIEW

**Comment by claudioshg** [comment]
Testing when build is up

**Comment by hifi-gustavo** [comment]
Build console output for commit: dd8f095bd8463a157737ae20ab87192c5587429e available at: https://gist.github.com/ea4c1b360c67750f01eb6152b9d16733<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9612-dd8f095.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9612-dd8f095.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0c28776ea2f3fd494d9d0e6c3df9f1cf6d0b7309 available at: https://gist.github.com/ea4c1b360c67750f01eb6152b9d16733<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9612-0c28776.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9612-0c28776.dmg

**Comment by claudioshg** [comment]
@druiz17 Can you explain a little more about 'Test 1'. Clicking the home button should change what?

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift + Touch and in Desktop Mode

Tested Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9612-0c28776.exe

Test 1: **PASS**
- Click users on use on the tablet and the home button at the bottom should change.

Test 2: **PASS**
- Click menu on the tablet then click on a submenu, then click the menu name at the top, which brings you back to the previous menu( in this test the main menu). The name on top should say "Menu" not
"root"

**NOTE**: 
All testing was done in VR and in Desktop mode. No problems whatsoever. 

I wasn't sure I understood what to do in Test 1. What I did was, click on 'Users' and once it took me to the screen that shows who's online, I clicked on the 'Home' button at the bottom of the tablet. It took me back to the main starting screen. If I misunderstood please let me know asap and I will re-test immediately. As of now and my understanding, this PR is a pass.

**Comment by druiz17** [comment]
For test1 the texture at the home button should change when you go to the home button.

**Comment by claudioshg** [comment]
I will check this^^

**Comment by claudioshg** [comment]
Test 1: PASS
The HOME icon does change texture from the '3x3 squares' icon to the X icon.
Tested with Oculus VR/Touch and in Desktop mode.

File: interface/resources/qml/hifi/tablet/TabletMouseHandler.qml [hunk 1]
```diff
@@ -87,7 +87,11 @@ Item {
                 if (topMenu.objectName === "") {
                     breadcrumbText.text = "Menu";
                 } else {
-                    breadcrumbText.text = topMenu.objectName;
+                    if (menuStack.length === 1) {
```
Comment (Atlante45) [inline interface/resources/qml/hifi/tablet/TabletMouseHandler.qml]: You could put this in the above conditional:
```cpp
if (topMenu.objectName === "" || menuStack.length === 1) {
    breadcrumbText.text = "Menu";
} else {
    breadcrumbText.text = topMenu.objectName;
}
```

Comment (sethalves) [inline interface/resources/qml/hifi/tablet/TabletMouseHandler.qml]: sorry clement I merged it while you were reading.  We'll get this in another PR

Comment (Atlante45) [inline interface/resources/qml/hifi/tablet/TabletMouseHandler.qml]: No worries, wasn't really blocking.

**Pull request merged**